pybossa/cache/task_browse_helpers.py

```diff
@@ -260,7 +260,7 @@ def parse_tasks_browse_args(args):
         if not isinstance(display_info_columns, list):
             display_info_columns = []
         parsed_args['display_info_columns'] = display_info_columns
-
+        allowed_fields.update({col.lower(): " task.info->>'{}'".format(col) for col in display_info_columns})
```

Comment:
It may be helpful to include a comment here on what the code is performing (allowing dynamic columns to be sortable, etc.). Thank you.

Reply:
added it!